Makefile

```diff
@@ -180,18 +181,12 @@ $(GOIMPORTS):
 
 .PHONY: build-cross
 build-cross: LDFLAGS += -extldflags "-static"
-build-cross: $(GOX)
-	GOFLAGS="-trimpath" CGO_ENABLED=0 $(GOX) -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/$(BINNAME)" -osarch='$(TARGETS)' $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' ./cmd/helm
+build-cross: $(GORELEASER)
+	LDFLAGS='$(LDFLAGS)' $(GORELEASER) build --snapshot --clean
 
 .PHONY: dist
 dist:
-	( \
-		cd _dist && \
-		$(DIST_DIRS) cp ../LICENSE {} \; && \
-		$(DIST_DIRS) cp ../README.md {} \; && \
-		$(DIST_DIRS) tar -zcf helm-${VERSION}-{}.tar.gz {} \; && \
-		$(DIST_DIRS) zip -r helm-${VERSION}-{}.zip {} \; \
-	)
+	LDFLAGS='$(LDFLAGS)' $(GORELEASER) release --snapshot --clean
```

Comment:
We build "canary" archives on every merge to main. You can see how that's built at https://github.com/helm/helm/blob/bdc459d73c44c6dad06289b49803cb1b3b4e21b7/.github/workflows/release.yml#L96-L99. 

You can read about downloading those at https://helm.sh/docs/intro/install/#from-canary-builds.

This change doesn't keep the same naming convention so the canary build links change. Is it possible to fix that to keep them as they were?

Also, `goreleaser release` does the build and packages up the release. In CI we run both which means the builds are happening twice. Can you update the github actions?